cypress/e2e/marc/marc-authority/edit-linked-authority-record/marc-authority-edit-151-field-that-controls-marc-bib.cy.js

```diff
@@ -0,0 +1,175 @@
+
+      cy.createTempUser([
+        Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+        Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+        Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+        Permissions.uiQuickMarcQuickMarcAuthorityLinkUnlink.gui,
+      ]).then((createdUserProperties) => {
+        testData.userProperties = createdUserProperties;
+
+        cy.login(testData.userProperties.username, testData.userProperties.password, {
+          path: TopMenu.marcAuthorities,
+          waiter: MarcAuthorities.waitLoading,
+        });
+      });
+    });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Users.deleteViaApi(testData.userProperties.userId);
+    InventoryInstance.deleteInstanceViaApi(createdRecordIDs[0]);
+    MarcAuthority.deleteViaAPI(createdRecordIDs[1]);
+  });
+
+  it(
+    'C374145 Edit tag value ("151") in the "MARC Authority" record which controls "MARC Bib(s)" (spitfire) (TaaS)',
+    { tags: ['extendedPath', 'spitfire'] },
+    () => {
+      MarcAuthorities.searchBy(testData.searchOption, marcFiles[1].authorityHeading);
+
+      MarcAuthorities.checkAuthorizedReferenceColumn(testData.authorized, testData.reference);
+      MarcAuthorities.selectItem(marcFiles[0].instanceTitle, false);
+
+      MarcAuthority.edit();
+      QuickMarcEditor.checkContent(testData.authority151FieldValue, testData.tag151RpwIndex);
+
+      QuickMarcEditor.updateExistingTagName(testData.tag151, testData.tag100);
+      QuickMarcEditor.pressSaveAndClose();
+      QuickMarcEditor.checkCallout(testData.cannotChangeCalloutMessage);
+      QuickMarcEditor.closeCallout();
+
+      QuickMarcEditor.updateExistingTagName(testData.tag100, testData.tag110);
+      QuickMarcEditor.pressSaveAndClose();
+      QuickMarcEditor.checkCallout(testData.cannotChangeCalloutMessage);
+      QuickMarcEditor.closeCallout();
+
+      QuickMarcEditor.updateExistingTagName(testData.tag110, testData.tag111);
+      QuickMarcEditor.pressSaveAndKeepEditing(testData.cannotChangeCalloutMessage);
+      QuickMarcEditor.closeCallout();
+
+      QuickMarcEditor.updateExistingTagName(testData.tag111, testData.tag130);
+      QuickMarcEditor.pressSaveAndClose();
+      QuickMarcEditor.checkCallout(testData.cannotChangeCalloutMessage);
+      QuickMarcEditor.closeCallout();
+
+      QuickMarcEditor.updateExistingTagName(testData.tag130, testData.tag150);
+      QuickMarcEditor.pressSaveAndClose();
+      QuickMarcEditor.checkCallout(testData.cannotChangeCalloutMessage);
+      QuickMarcEditor.closeCallout();
+
+      QuickMarcEditor.updateExistingTagName(testData.tag150, testData.tag155);
+      QuickMarcEditor.pressSaveAndKeepEditing(testData.cannotChangeCalloutMessage);
+      QuickMarcEditor.closeCallout();
+
+      QuickMarcEditor.updateExistingTagName(testData.tag155, testData.tag152);
+      // Todo: the below two lines should be uncommented once https://issues.folio.org/browse/UIQM-526 is resolved.
```

Comment:
Please remove comments

Reply:
There is an issue on step 11. Actual result and expected results are different. So, if the comments are removed, the test will fail. Should I remove all these comments anyway?

Reply:
No need to remove comments, thanks for clarification